### PR TITLE
Normalize email addresses to NFC

### PR DESCRIPTION
**Pull request by ugexe**

### CODE REVIEW

File: fixtures/insights.json [hunk 1]
```diff
@@ -136,7 +136,6 @@
           "autonomous_system_organization": "Linkem IR WiMax Network",
           "connection_type": "Cable/DSL",
           "domain": "example.com",
-          "ip_address": "1.2.3.4",
```
Comment (oschwald) [inline fixtures/insights.json:139]: Why was this removed?

**Review by oschwald — CHANGES_REQUESTED** [review]

File: fixtures/insights.json (hunk 1 above)
Comment (ugexe) [reply fixtures/insights.json:139]: The "Add json linter" commit caught the duplicate `ip_address` field in this object. I kept the second instance of this field since that should be the one ultimately getting used after the first is overridden.

Comment (oschwald) [reply fixtures/insights.json:139]: Ahh, that makes sense. I should have looked more closely.

**Review by oschwald — APPROVED** [review]
